calculator/basic/Calculator.java

```diff
@@ -0,0 +1,63 @@
+package calculator.basic;
+
+public class Calculator {
+   
+    private double result;
+
+    public Calculator () {
+        this.result = 0;
+    }    
+
+    public double getResult() {
+        return this.result;
+    }
+
+    public String toString() {
+
+        if (result == (int)result) {
```

Comment:
decide if you are using 'this.result' or just 'result' inside class methods and be consistent.